src/pytest_elasticsearch/factories.py

```diff
@@ -42,7 +42,7 @@ def return_config(request):
     for option in options:
         option_name = 'elasticsearch_' + option
         conf = request.config.getoption(option_name) or \
-            request.config.getini(option_name)
+               request.config.getini(option_name)
```

Comment:
Do not change unrelated lines. Especially if something is just a formatting difference.

I plan to employ black to actually keep an eye on the format, but until then, if it passes pylint and pycodestyle, do not reformat it if you're not changing it.

Reply:
Didnt mean this. Will update now.